spring-integration-jms/src/test/java/org/springframework/integration/jms/JmsOutboundInsideChainTests.java

```diff
@@ -36,11 +36,12 @@
  * //INT-2275
  *
  * @author Artem Bilan
+ * @author Gary Russell
  */
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration
 @DirtiesContext
-public class JmsOutboundInsideChainTests {
+public class JmsOutboundInsideChainTests extends ActiveMQMultiContextTests {
```

Comment:
Yeah... The same here.

Or does that `new ActiveMQConnectionFactory("vm://localhost?broker.persistent=false")` really mean this:

> to keep a single broker up

and that already doesn't matter how much `new ActiveMQConnectionFactory("vm://localhost?broker.persistent=false")` we are going to call yet?

Reply:
Exactly - `vm://...` will connect to an existing broker, or start a new one.